examples/lighting-app/nrfconnect/main/AppTask.cpp

```diff
@@ -368,6 +394,21 @@ void AppTask::StartBLEAdvertisementHandler(AppEvent * aEvent)
     }
 }
 
+void AppTask::ThreadProvisioningHandler(const ChipDeviceEvent * event, intptr_t arg)
+{
+    (void) arg;
+    if ((event->Type == DeviceEventType::kServiceProvisioningChange) && ConnectivityMgr().IsThreadProvisioned())
+    {
+        #ifdef CONFIG_CHIP_NFC_COMMISSIONING
```

Comment:
I would either remove tabs and start #ifdef from the beginning of the line, or move it before if statement in line 400.

Reply:
I didn't do this intentionally, because I thought that in the future we may use this handler for performing some other actions than disabling NFC, as Thread provisioned event is not related only to NFC control. Anyway @Damian-Nordic wrote the same, so I will do as you want for now and maybe someday we will decide to extend this handler by other actions.